### PR TITLE
Add LB feature module

### PR DESCRIPTION
**Pull request by jamiebrynes7**
Split from #1491, this contains a basic LB implementation that satisfies the Playground, FPS, and Blank project

### CODE REVIEW

File: workers/unity/Packages/io.improbable.gdk.loadbalancing/LoadBalancingHelper.cs [hunk 1]
```diff
@@ -16,38 +17,29 @@ public static void AddLoadBalancingSystems(this WorkerInWorld worker, Action<Loa
...
+            World.GetOrCreateSystem<ClassifyWorkersSystem>();
         }
 
-        public void AddPartitionManagement(params string[] workerTypes)
+        public void AddPartitionManagement(string workerType, params string[] workerTypes)
         {
-            var partitionManagementSystem = world.GetOrCreateSystem<PartitionManagementSystem>();
-            partitionManagementSystem.WorkerTypes = workerTypes;
+            var partitionManagementSystem = World.GetOrCreateSystem<PartitionManagementSystem>();
+            partitionManagementSystem.WorkerTypes = workerTypes.Append(workerType).ToArray();
```
Comment (zeroZshadow) [inline workers/unity/Packages/io.improbable.gdk.loadbalancing/LoadBalancingHelper.cs:31]: If we pass in no additional `workerTypes`, is it null or an empty array?

**Review by zeroZshadow — APPROVED** [review]

File: workers/unity/Packages/io.improbable.gdk.loadbalancing/LoadBalancingHelper.cs (hunk 1 above)
Comment (jamiebrynes7) [reply workers/unity/Packages/io.improbable.gdk.loadbalancing/LoadBalancingHelper.cs:31]: [Empty array](https://stackoverflow.com/questions/2942050/passing-zero-arguments-as-params-where-the-behaviour-is-defined)

**Review by paulbalaji — APPROVED** [review]

File: workers/unity/Packages/io.improbable.gdk.loadbalancing/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "io.improbable.gdk.loadbalancing",
+  "displayName": "SpatialOS GDK Load Balancing",
+  "version": "0.4.0",
+  "unity": "2019.3",
```
[the remark this answers is not in the document]
Comment (jamiebrynes7) [reply workers/unity/Packages/io.improbable.gdk.loadbalancing/package.json:5]: Given I copied this from another package.. I would guess so.

If we use 2020.x only features... we should definitely change this across the board

[the remark this answers is not in the document]
Comment (jamiebrynes7) [reply workers/unity/Packages/io.improbable.gdk.loadbalancing/package.json:5]: Will add this to the todo list to follow up on after these changes

[the remark this answers is not in the document]
Comment (paulbalaji) [reply workers/unity/Packages/io.improbable.gdk.loadbalancing/package.json:5]: 👍

**Pull request merged**